apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/ListBucketsTest.cs

```diff
@@ -75,6 +75,58 @@ public void PartialResponses()
             }
         }
 
+        [Fact]
+        public async Task SoftDeletedOnly()
+        {
+            var bucketName = _fixture.GenerateBucketName();
+            var softDeleteBucket = _fixture.CreateBucket(
+                bucketName,
+                multiVersion: false,
+                softDelete: true
+            );
+            await _fixture.Client.DeleteBucketAsync(
+                softDeleteBucket.Name,
+                new DeleteBucketOptions { DeleteObjects = true }
+            );
+            var actualBuckets = await _fixture
+                .Client.ListBucketsAsync(
+                    _fixture.ProjectId,
+                    new ListBucketsOptions { SoftDeletedOnly = true }
+                )
+                .ToListAsync();
```

Comment:
Lots of unnecessary wraping here:

```suggestion
            var softDeleteBucket = _fixture.CreateBucket(bucketName, multiVersion: false, softDelete: true);
            
            await _fixture.Client.DeleteBucketAsync(softDeleteBucket.Name, new DeleteBucketOptions { DeleteObjects = true });
            
            var actualBuckets = await _fixture.Client
                .ListBucketsAsync(_fixture.ProjectId, new ListBucketsOptions { SoftDeletedOnly = true })
                .ToListAsync();
```